lib/spack/spack/cmd/checksum.py

```diff
@@ -71,3 +75,35 @@ def checksum(parser, args):
     print()
     print(version_lines)
     print()
+
+    if args.add_to_package:
+        filename = spack.repo.path.filename_for_package_name(args.package)
+        # Make sure we also have a newline after the last version
+        versions = [v + '\n' for v in version_lines.splitlines()]
+        versions.insert(0, '    # FIXME: Added by `spack checksum`\n')
```

Comment:
Why not add this with `append` after we reverse the list?

Reply:
Done